File: BankingTool.Model/Dto/RequestDtos/TransactionsListForCustomerRequest.cs
namespace BankingTool.Model.Dto.RequestDtos
{
    public class TransactionsListForCustomerRequestObject
    {
        public TransactionsListForCustomerRequest Request { get; set; }
    }
    public class TransactionsListForCustomerRequest
    {
        public TransactionsListForCustomerRequestCustomer Customer { get; set; }
        public TransactionsListForCustomerRequestAccount Account { get; set; }
        public TransactionsListForCustomerRequestBank Bank { get; set; }
    }
    public class TransactionsListForCustomerRequestBank : RequestId
    {
    }
    public class TransactionsListForCustomerRequestAccount : RequestId
    {
    }
    public class TransactionsListForCustomerRequestCustomer : RequestId
    {
    }
}

File: bankingtool.client/src/app/dto/request/bank-account/transactions-list-for-customer-request.ts
import { RequestId } from "../request-id";

export interface TransactionsListForCustomerRequestObject {
  request: TransactionsListForCustomerRequest;
}

export interface TransactionsListForCustomerRequest {
  customer: TransactionsListForCustomerRequestCustomer;
  account: TransactionsListForCustomerRequestAccount;
  bank: TransactionsListForCustomerRequestBank;
}

export interface TransactionsListForCustomerRequestBank extends RequestId {

}

export interface TransactionsListForCustomerRequestAccount {
  accountTypeId: number;
}

export interface TransactionsListForCustomerRequestCustomer extends RequestId {

}
